src/main/kotlin/no/nav/familie/ks/sak/internal/kontantstøtteOvergangsordningBrevNovember2024/SendInformasjonsbrevOvergangsordningNov2024Task.kt

```diff
@@ -0,0 +1,65 @@
+package no.nav.familie.ks.sak.internal.kontantstøtteOvergangsordningBrevNovember2024
+
+import no.nav.familie.ks.sak.api.dto.ManueltBrevDto
+import no.nav.familie.ks.sak.common.exception.Feil
+import no.nav.familie.ks.sak.integrasjon.pdl.PersonopplysningerService
+import no.nav.familie.ks.sak.internal.kontantstøtteOvergangsordningBrevNovember2024.SendInformasjonsbrevOvergangsordningNov2024Task.Companion.TASK_STEP_TYPE
+import no.nav.familie.ks.sak.kjerne.brev.BrevService
+import no.nav.familie.ks.sak.kjerne.brev.domene.maler.Brevmal
+import no.nav.familie.ks.sak.kjerne.fagsak.FagsakService
+import no.nav.familie.ks.sak.kjerne.personopplysninggrunnlag.domene.Målform
+import no.nav.familie.prosessering.AsyncTaskStep
+import no.nav.familie.prosessering.TaskStepBeskrivelse
+import no.nav.familie.prosessering.domene.Task
+import no.nav.familie.prosessering.internal.TaskService
+import org.springframework.stereotype.Service
+
+@Service
+@TaskStepBeskrivelse(
+    taskStepType = TASK_STEP_TYPE,
+    beskrivelse = "Send informasjonsbrev om kontantstøtteendring",
+    maxAntallFeil = 1,
+)
+class SendInformasjonsbrevOvergangsordningNov2024Task(
+    private val brevService: BrevService,
+    private val fagsakService: FagsakService,
+    private val personopplysningerService: PersonopplysningerService,
+    private val taskService: TaskService,
+) : AsyncTaskStep {
+    override fun doTask(task: Task) {
+        val alleTaskerMedSammePayloadOgType =
+            taskService.finnAlleTaskerMedPayloadOgType(payload = task.payload, type = task.type)
+
+        if (alleTaskerMedSammePayloadOgType.size > 1) {
```

Comment:
Er sjekken nødvendig? Er det fare for duplikate tasker? Burde i alle fall filterere vekk avvikshåndterte

Reply:
Husk også at tasker som er FERDIG autoslettes etter 4 uker.

Reply:
Vi tenkte å kjøre på med noen få først, så det er greit med denne når vi kjører hele runden